timemachine/fe/free_energy.py

```diff
@@ -898,6 +905,61 @@ def compute_intermediate_result(lambdas: Sequence[float]) -> PairBarResult:
     return results, trajectories
 
 
+def compute_potential_matrix(
+    potential: custom_ops.Potential,
+    hrex: HREX[CoordsVelBox],
+    params_by_state: NDArray,
+    max_delta_states: Optional[int] = None,
+) -> NDArray:
+    """Computes the (n_replicas, n_states) sparse matrix of potential energies, where a given element $(k, l)$ is
+    computed iff state $l$ is within `max_delta_states` of the current state of replica $k$, and is otherwise set to
```

Comment:
```suggestion
    computed if state $l$ is within `max_delta_states` of the current state of replica $k$, and is otherwise set to
```

Reply:
This was intended as an abbreviation for "if and only if". Removed the abbreviation in https://github.com/proteneer/timemachine/pull/1309/commits/99fc96d6c939ec669cdd37d029e074032a8b4c7f for clarity.

Reply:
Ah, that would have been fine actually, just read it as a typo.